packages/api/tests/lib/test-rulesHelpers.js

```diff
@@ -210,6 +210,13 @@ test('queueMessageForRule includes eventSource in payload, if provided', async (
   t.deepEqual(payload.meta.eventSource, eventSource);
 });
 
+test('queueMessageForRule includes queueUrl in rule, if provided', async (t) => {
```

Comment:
This proves that the value of `rule.queueUrl` gets passed onto the helper that queues messages for SQS/Kinesis rules, however given the mocking here and that this is a unit test, it doesn't fully prove the feature is functional. 

Should we add an integration test?

Reply:
I should add that I'm not sure how an integration test would work. How can we easily verify that the execution was started from a given queue?

Reply:
Maybe this would work... In `packages/integration-tests/index.js` you can use `getExecutionInput` and check the queue URL? @markdboyd

Reply:
That would allow me to inspect the `queueUrl` in the execution input message, but there's no guarantee (that I'm aware of) that the `queueUrl` in the message is **actually** the queue that the message was read from. They **should** always be the same, but I'm not sure if we can rely on it